src/db/clients/sqlserver.js

```diff
@@ -184,6 +185,16 @@ export async function listTableTriggers(conn, table) {
   return data.map((row) => row.trigger_name);
 }
 
+export async function listTableIndexes(conn, table) {
+  // SQL Server does not have information_schema for indexes, so other way around
+  // is using sp_helpindex stored procedure to fetch indexes related to table
+  const sql = `EXEC sp_helpindex ${wrapIdentifier(table)}`;
```

Comment:
I removed the `wrapIdentifier`, but i could not use it as as query param like in mysql.
That is not implemented in this sqlserver client. If you take a look in the other functions, they are all like this. Because the execute just get the query string and execute. It does not prepare it by anyways.
`export async function driverExecuteQuery(conn, queryArgs) {
  const runQuery = async (connection) => {
    const request = connection.request();
    if (queryArgs.multiple) {
      request.multiple = true;
    }

    return {
      request,
      data: await request.query(queryArgs.query),
    };
  };

  return conn.connection
    ? runQuery(conn.connection)
    : runWithConnection(conn, runQuery);
}`

It is possible to make that better ans use the PreparedStatement of sqlserver drive, but i think that it would go better in other issue, that i can even help on.